Makefile

```diff
@@ -388,9 +388,9 @@ docs/%: export TFDOCS_RECURSIVE ?= $(if $(wildcard $(TFDOCS_PATH)/modules),--rec
 docs/%: export TFDOCS_TEMPLATE ?= <!-- BEGIN TFDOCS -->\n{{ .Content }}\n\n<!-- END TFDOCS -->
 docs/%: export TFDOCS_FILE ?= README.md
 docs/%: export TFDOCS_PATH ?= .
-docs/%: export README_FILES ?=  $(if $(wildcard $(TFDOCS_FILE)),true,)
-docs/%: export TF_FILES ?= $(if $(wildcard $(TFDOCS_PATH)/*.tf),true,)
-docs/%: export HCL_FILES ?= $(if $(wildcard $(TFDOCS_PATH)/*.pkr.hcl),true,)
+docs/%: export README_FILES ?=  $(if $(wildcard $(TFDOCS_FILE)),true,$(if $(wildcard modules/*/$(TFDOCS_FILE)),true,))
+docs/%: export TF_FILES ?= $(if $(wildcard $(TFDOCS_PATH)/*.tf),true,$(if $(wildcard modules/*/*.tf),true,))
+docs/%: export HCL_FILES ?= $(if $(wildcard $(TFDOCS_PATH)/*.pkr.hcl),true,$(if $(wildcard /modules/*/*.pkr.hcl),true,))
```

Comment:
The structure for packer projects is a bit different. I don't think we need to make this change at this time...

Reply:
I reverted line 393.